code/src/App.js

```diff
@@ -1,12 +1,33 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import Album from "./components/Album";
+import Header from "./components/Header";
 
-console.log(data)
+console.log(data);
+
+// Albums array
+const albums = data.albums.items;
+console.log(albums);
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <Header />
+
+      <div className="album-container">
+        {albums.map((album) => {
```

Comment:
It would be more readable if you use `albums.map((album) => ( ) )` to avoid using return nested in another return